website/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "scripts": {
-    "prestart": "cp ../docs/cli.md.template ../docs/cli.md && node ../cli.js markdown-help >> ../docs/cli.md",
+    "prestart": "cp ../docs/cli.md.template ../docs/cli.md && node ../cli.js markdown-help >> ../docs/cli.md && cp -r ../dist/katex.min.css ../dist/fonts static/static",
```

Comment:
Instead of copying files, we can use build artifacts from Circle CI build. However this needs a CORS proxy.

Reply:
Using build artifacts from Circle CI sounds nice.  What's involved with a CORS proxy?